<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManufacturerDialog</class>
 <widget class="QDialog" name="ManufacturerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>344</width>
    <height>284</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Производитель</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QListWidget" name="listWidget">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="uniformItemSizes">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLineEdit" name="lineEdit">
     <property name="placeholderText">
      <string>Новый производитель</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="addButton">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="asm.qrc">
       <normaloff>:/button/pictures/plus.png</normaloff>:/button/pictures/plus.png</iconset>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="asm.qrc"/>
 </resources>
 <connections/>
</ui>
